.github/workflows/CI.yml

```diff
@@ -181,4 +181,6 @@ jobs:
         pytest -vx tests/test_num_params.py
         pytest -vx tests/test_param_shapes.py
         pytest -vx tests/test_param_types.py
-        pytest -vx tests/test_ssim.py
\ No newline at end of file
+        pytest -vx tests/test_ssim.py
+        pytest -vx tests/submission_runner_test.py
```

Comment:
do we even need the `test_submission` function in this file anymore? it seems like that's covered by test_baselines.py now

Reply:
I suppose we don't I removed it.